.github/codecov.yaml

```diff
@@ -0,0 +1,50 @@
+# Find more at https://docs.codecov.com/docs/codecovyml-reference
+codecov:
+  bot: "Codecov Bot"
+  max_report_age: 12
+  require_ci_to_pass: yes
+  notify:
+    after_n_builds: 1
+    wait_for_ci: yes
+
+# Layout of the PR comment produced by Codecov bot
+comment:
+  layout: "header, diff, flags, components, files"
+
+flag_management:
+  individual_flags:
+    - name: unit
+      paths:
+        - pkg/**
```

Comment:
The api directory, at the moment, includes some vendor code that we needed to included that way instead of a dependency.. so you'll need to exclude `api/external` because of [external/maistra](https://github.com/Kuadrant/kuadrant-operator/tree/main/api/external/maistra)

Reply:
Will do! Thanks!